.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Please propose features in the Discussions area as an `idea`**
```

Comment:
Can we instead just redirect them to the discussions page?

Reply:
I was hoping we could do something like that but I do not know much about configuring Github in that way.